### ISSUE TITLE
Option to Compare Up to Four Colleges

### ISSUE THREAD

**Issue by maurahoward** [forsythetony/CollegeComparison#29]
Tony, would it be possible to compare up to four colleges side-by-side? I think this would make the compare feature/app as a whole a lot stronger. From what I'm seeing, it looks like your design has left room for this too.

**Comment by forsythetony** [comment]
Shouldn't be too hard, but I'd have to see how it looks. I'm just worried it might look a little crammed.

**Comment by maurahoward** [comment]
Yeah i am a fan of how clean it looks now. Try making the bars more narrow and then even putting the info tab on the left side of the screen (the y axis) versus the bottom. 

Also, and we may want additional feedback on this, I don't think we need the word "info" above that tab. A few lines that indicate a "pull up" swipe is pretty common with iOS now -- so common it's even on the iOS 5 lock screen.

**Comment by forsythetony** [comment]
I took out the "info" above the tab but I'm not sure if we should move it to the left. It might interfere with the user swiping to navigate through pages.

**Comment by maurahoward** [comment]
Oh I didn't even think of that. Have you tried with four bars plus the info bar? I think it would work. Here's what I drew up.

![screen shot 2014-01-13 at 10 28 55 pm](https://f.cloud.github.com/assets/6326457/1907618/744f58a8-7cd4-11e3-834b-926d961da74a.png)

**Comment by maurahoward** [comment]
Actually don't have the info bar as thin as the one above, that looks bad. More like this: 

![screen shot 2014-01-13 at 10 34 26 pm](https://f.cloud.github.com/assets/6326457/1907632/44f95b52-7cd5-11e3-9128-a612b2f64e06.png)

**Comment by forsythetony** [comment]
Yea that should work just fine. I am a bit worried about how the names would fit. Maybe we could assign a color to each college? At any rate, it should be pretty straightforward to code there's just a lot I'd have to change, but give me a couple days and I'll have it done.
